server/models/handlers/workerTimeslot.js

```diff
@@ -10,6 +11,13 @@ async function addWorkerTimeslot(slotId, schoolId, workerId, status, date) {
         return false;
     }
 };
+async function checkWorkerAvailability(workerTimeslotId) {
+    if (_.isNil(workerTimeslotId)) return false;
+    if (!_.isNumber(workerTimeslotId)) return false;
```

Comment:
HI Nancy,
Due to the nature of the request, it is likely that a string value of number may be passed down from the UI, e.g., `'3'` instead of `3`. The _.isNumber lodash method will return false for string values of valid numbers (as shown below). Thus, I would recommend removing this condition. In the event that a non-numeric value was passed down, the query will return nothing and thus, false will automatically be returned. As a result, I would recommend removing this condition to avoid potential issues.
![image](https://user-images.githubusercontent.com/32720322/100937117-86828d00-34c0-11eb-8073-812d56fb7d13.png)